source/features/infinite-scroll.js

```diff
@@ -26,6 +25,10 @@ const loadMore = debounce(() => {
 const inView = new IntersectionObserver(([{isIntersecting}]) => {
 	if (isIntersecting) {
 		loadMore();
+	} else {
+		// The button may have been changed after its gone out of view
+		// So try finding it again
+		findButton();
```

Comment:
??

Reply:
I've tested the feature on user dashboard, and have found that the callback executed with `isIntersecting` values in the following manner.

``` js
false // initial page load
true  // regular intersection on scroll down
false // immediate callback as button is pushed down the viewport
// no more callbacks
```

But after the third callback, the button is being replaced with a new one, so the intersection observer stops firing events because the old button doesn't exist in the DOM anymore.

You can verify this behavior by logging the `btn` var inside the callback and hovering over it in the console to verify if it is still in the DOM!

Reply:
That's weird, `findButton` is already called by `observeEl` when the new list (and allegedly the button) is added to the DOM. If that doesn't work anymore, then `observeEl` should probably be dropped.

Reply:
For some reason the mutation callback is not being called even with `childList` and `subtree` set to `true`.

I've tested it on Chrome 70, and even Canary 73 to be sure.

Reply:
Let’s replace it with a single `findButton` call then

Reply:
Yup, updated it already!

Reply:
Ah yeah I didn’t see the commit ☺️